pkg/utils/nodeclass/nodeclass.go

```diff
@@ -72,7 +72,7 @@ func NewSubnetSelectorTerms(subnetSelector map[string]string) (terms []v1beta1.S
 	for k, v := range subnetSelector {
 		switch k {
 		case "aws-ids", "aws::ids":
-			ids = strings.Split(strings.Trim(v, " "), ",")
+			ids = strings.Split(strings.Replace(v, " ", "", -1), ",")
```

Comment:
Shouldn't we just do the ReplaceAll instead https://pkg.go.dev/strings#ReplaceAll